Ollivanders/src/net/pottercraft/Ollivanders2/Potion/IngredientType.java

```diff
@@ -7,35 +7,51 @@
 
 public enum IngredientType
 {
+   ACONITE (Material.SPIDER_EYE, (short)0, "Aconite"),
    ARMADILLO_BILE (Material.POTION, (short)9, "Armadillo Bile"),
+   ASPHODEL (Material.DEAD_BUSH, (short)0, "Asphodel"),
    BEZOAR (Material.COAL, (short)1, "Bezoar"), // charcoal
    BILLYWIG_STING_SLIME (Material.SLIME_BALL, (short)0, "Billywig Sting Slime"),
    BOOM_BERRY_JUICE (Material.POTION, (short)11, "Boom Berry Juice"),
+   BOOMSLANG_SKIN (Material.ROTTEN_FLESH, (short)0, "Boomslang Skin"),
    BONE (Material.BONE, (short)0, "Bone"),
    BLOOD (Material.POTION, (short)7, "Blood"),
    CHIZPURFLE_FANGS (Material.PUMPKIN_SEEDS, (short)0, "Chizpurfle Fangs"),
    DEATHS_HEAD_MOTH_CHRYSALIS (Material.COAL, (short)0, "Death's Head Moth Chrysalis"),
    DEW_DROP (Material.GHAST_TEAR, (short)0, "Dew Drop"),
    DITTANY (Material.SAPLING, (short)2, "Dittany"), //birch sapling
+   DRAGON_BLOOD (Material.SPLASH_POTION, (short)1, "Dragon Blood"),
+   DRAGONFLY_THORAXES (Material.BEETROOT_SEEDS, (short)0, "Dragonfly Thoraxes"),
    DRIED_NETTLES (Material.SAPLING, (short)0, "Dried Nettles"), // oak sapling
+   FAIRY_WING (Material.GOLD_NUGGET, (short)0, "Fairy Wing"),
+   FIRE_SEEDS (Material.NETHER_WARTS, (short)0, "Fire Seeds"),
    FLOBBERWORM_MUCUS (Material.SLIME_BALL, (short)0, "Flobberworm Mucus"),
+   FLUXWEED (Material.DEAD_BUSH, (short)0, "Fluxweed"),
+   FULGURITE (Material.GLOWSTONE_DUST, (short)0, "Fulgurite)"),
    GALANTHUS_NIVALIS (Material.RED_ROSE, (short)3, "Galanthus Nivalis"), //azure bluet
    GINGER_ROOT (Material.BEETROOT, (short)0, "Ginger Root"),
+   GROUND_DRAGON_HORN (Material.GLOWSTONE_DUST, (short)0, "Ground Dragon Horn"),
    GROUND_PORCUPINE_QUILLS (Material.INK_SACK, (short)3, "Ground Porcupine Quills"),
    GROUND_SCARAB_BEETLE (Material.SULPHUR, (short)0, "Ground Scarab Beetle"),
    GROUND_SNAKE_FANGS (Material.INK_SACK, (short)15, "Ground Snake Fangs"),
    HONEYWATER (Material.POTION, (short)0, "Honeywater"),
    HORKLUMP_JUICE (Material.DRAGONS_BREATH, (short)0, "Horklump Juice"),
    HORNED_SLUG_MUCUS (Material.SLIME_BALL, (short)0, "Horned Slug Mucus"),
+   HORN_OF_BICORN (Material.BLAZE_ROD, (short)0, "Horn of Bicorn"),
    INFUSION_OF_WORMWOOD (Material.POTION, (short)5, "Infusion of Wormwood"),
    JOBBERKNOLL_FEATHER (Material.FEATHER, (short)0, "Jobberknoll Feather"),
+   Knotgrass (Material.LONG_GRASS, (short)0, "Knotgrass"),
+   LACEWING_FLIES (Material.PUMPKIN_SEEDS, (short)0, "Lacewing Flies"),
    LAVENDER_SPRIG (Material.DOUBLE_PLANT, (short)1, "Lavender Sprig"), //lilac
+   LEECHES (Material.INK_SACK, (short)0, "Leeches"),
    LETHE_RIVER_WATER (Material.POTION, (short)0, "Lethe River Water"), //bottle of water
    LIONFISH_SPINES (Material.RAW_FISH, (short)0, "Lionfish Spines"),
    MANDRAKE_LEAF (Material.WATER_LILY, (short)0, "Mandrake Leaf"),
+   MERCURY (Material.INK_SACK, (short)0, "Mercury"),
```

Comment:
Shouldn't this be a silver liquid?

Reply:
Yeah. Maybe a Gahst tear? This was also from my Philosophers stone research so it can be removed 
,.

Reply:
Yea or silver colored liquid in a glass bottle like you did with blood just with color 13

Reply:
Yea or silver colored liquid in a glass bottle like you did with blood just with color 13

Reply:
Made this a bottle of silver liquid